Formula/ocrmypdf.rb

```diff
@@ -21,7 +21,6 @@ class Ocrmypdf < Formula
   depends_on "jbig2enc"
   depends_on "jpeg"
   depends_on "leptonica"
-  depends_on "libffi"
```

Comment:
Still picking up linkage with brewed `libffi`:
```
❯ otool -L _cffi_backend.cpython-39-darwin.so
_cffi_backend.cpython-39-darwin.so:
        @@HOMEBREW_PREFIX@@/opt/libffi/lib/libffi.7.dylib (compatibility version 9.0.0, current version 9.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.60.1)
```
No point in trying to remove it from just this formula, as it currently has an indirect dependency on `libffi` anyway.

Reply:
Hmm, it looks like it's being pulled in as indirect dependency because `guile`, `p11-kit`, and `glib` all use `depends_on "libffi"`.  I think a cleanup of this is needed, though this should be done in a separate PR.